src/main/java/seedu/address/model/person/SalaryParserUtil.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Contains utility methods used for parsing strings and longs related to {@code Salary}.
+ */
+public class SalaryParserUtil {
+
+    /**
+     * Returns the salary value converted from a long to a string with 2 decimals.
+     */
+    public static String convertLongSalaryToString(long value) {
+        StringBuilder stringBuilder = new StringBuilder(Long.toString(value));
+
+        // ensure that string has at least (Salary.NUM_OF_DECIMAL_PLACES + 1) digits
+        // to insert the decimal point properly
+        if (stringBuilder.charAt(0) == '-') {
+            addZerosToFront(stringBuilder, 1);
+        } else {
+            addZerosToFront(stringBuilder, 0);
+        }
+
+        int decimalPlaceIndex = stringBuilder.length() - Salary.NUM_OF_DECIMAL_PLACES;
+        stringBuilder.insert(decimalPlaceIndex, ".");
+        return stringBuilder.toString();
+    }
+
+    /**
+     * Adds zeros to the front of the {@code stringBuilder} from index {@code start} until the string has at least
+     * ({@code Salary.NUM_OF_DECIMAL_PLACES} + 1) digits.
+     */
+    private static void addZerosToFront(StringBuilder stringBuilder, int start) {
```

Comment:
I think this function can be simplified unless I am misunderstanding the intend. But I think its fine to keep the current way too.

```
jshell> String val = "-123";
val ==> "-123"

jshell> String.format("%06d", Long.parseLong(val));
$13 ==> "-00123"
```

Reply:
Thank you for the suggestion! I have modified the function!